packages/code-du-travail-frontend/src/home/Outils.js

```diff
@@ -48,47 +48,42 @@ const outils = [
   }
 ];
 
-export default class Outils extends React.PureComponent {
-  static propTypes = {};
+const Outils = ({ title }) => (
```

Comment:
why did you move from PureComponent to function Component (without memo) ?

Reply:
I want to remove step by step all the classes of the project, and for this one I thought that PureComponent was not bringing much value since this component is pretty simple (https://stackoverflow.com/questions/46736983/will-a-stateless-component-re-render-if-its-props-have-not-changed) but I'm goind to add [React.memo](https://fr.reactjs.org/docs/react-api.html#reactmemo) if you're ok with it ?

Reply:
A quote from the stackoverflow answer linked above: `Important to note is that calling render() doesn't mean that DOM Nodes are being manipulated in any way. The render method just serves the diff algorithm to decide which DOM Nodes need to really be attached / detached. Note that render() is not expensive, it's the DOM manipulations that are expensive. They are executed only if render() returns different virtual trees.`